server/methods/loadHistory.js

```diff
@@ -7,7 +7,7 @@ import { settings } from '../../app/settings';
 import { loadMessageHistory } from '../../app/lib';
 
 Meteor.methods({
-	loadHistory(rid, end, limit = 20, ls) {
+	loadHistory(rid, end, limit = 20, ls, showThreadMessages = false) {
```

Comment:
just to mention that having `showThreadMessages = false` as default actually introduces a breaking change, since it will change the default result.. I'm still going to review everything else, but I'd like to point this out =)